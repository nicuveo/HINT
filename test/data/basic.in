package basic

import     "lib/foo"
import bar "lib/bar"

...
