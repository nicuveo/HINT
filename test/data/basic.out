package basic
import "lib/foo"
import bar "lib/bar"
a: b: 42
